policyengine_us/parameters/gov/states/mt/tax/income/social_security/rate1.yaml

```diff
@@ -0,0 +1,11 @@
+description: 
+values:
+  2021-01-01: 0.5
+
+metadata:
+  unit: /1
+  period: year
+  label: Montana social security benefits rate 1
+  reference:
+    - title: 2021 Montana Individual Income Tax Return 
+      href: https://mtrevenue.gov/wp-content/uploads/mdocs/form%202%202021.pdf#page=6
```

Comment:
add legal code reference and 2022 / 2023 and legal code references

Reply:
All of those numbers only appear in the tax form.